src/modules/python/pbs_hooks/PBS_power.PY

```diff
@@ -287,6 +287,31 @@ if this_event.type == pbs.PERIODIC:
                         exec_vnodes[vn]["neededby"] = start_time
         i += 1
 
+    try:
+        pbs_conf = pbs.get_pbs_conf()
+        pbs_home = pbs_conf['PBS_HOME']
```

Comment:
Don't put this in a try/except.  Just say if 'PBS_HOME' in pbs_conf

Reply:
Okay.